scripts/generate_bottle.sh

```diff
@@ -45,7 +45,6 @@ echo "class Eosio < Formula
    option :universal
 
    depends_on \"gmp\"
-   depends_on \"gettext\"
```

Comment:
What about `gettext-devel` in`.cicd/platforms/centos-7.6-unpinned.dockerfile` and `.cicd/platforms/centos-7.6.dockerfile`. 

If `.cicd` is not going to be maintained it should be removed.

Reply:
Yes absolutely -- anything not used or not tested ought to be removed. `.cicd`  & build scripts fall in that bucket now.

I've been reluctant to PR a change to zap all that since there really isn't a replacement for pinned builds outside the build script. But.. the build scripts are no longer tested or getting attention soooo :shrug:  :confused:

Reply:
There is some thought that the `.cicd` will be used by a new cicd system, which is what Matthew Darwin is talking about below. Maybe go ahead and make the those two changes so we have them just in case.

Reply:
imo, there is no viable path for `.cicd` as is: it's outdated, neglected, overcomplicated, interwoven with b1 infra, and of course currently completely untested. The best course of action appears to be to eliminate it and start anew with something else; using the old files as reference if need be.

Reply:
I'll buy that.